target/linux/ath79/generic/base-files/etc/board.d/02_network

```diff
@@ -639,6 +640,10 @@ ath79_setup_macs()
 		label_mac=$lan_mac
 		;;
 	dlink,dir-859-a1|\
+	dlink,dir-869-a1)
+                lan_mac=$(mtd_get_mac_text "devdata" 0xc9)
+                wan_mac=$(mtd_get_mac_text "devdata" 0x79)
+		;;
```

Comment:
Why are these devices use a different way to extract the mac addresses now?

Reply:
Hi, can you tell me more about what you mean please?

Reply:
This change also affects the `dlink,dir-859-a1` device. If the `dlink,dir-859-a1` is detected it will also go into your new condition. If this is not done intentional please change the code like this:
````
 		wan_mac=$(mtd_get_mac_ascii devdata "wanmac")
 		label_mac=$lan_mac
 		;;
+	dlink,dir-869-a1)
+		lan_mac=$(mtd_get_mac_text "devdata" 0xc9)
+		wan_mac=$(mtd_get_mac_text "devdata" 0x79)
+		;;
 	dlink,dir-859-a1|\
 	qihoo,c301|\
 	wd,mynet-n600|\
````

Please also use tabs for indention and not spaces.

Reply:
This is still open

Reply:
Thanks for the help, it should be fine now.